opm/material/fluidsystems/blackoilpvt/ConstantCompressibilityOilPvt.hpp

```diff
@@ -181,8 +181,8 @@ class ConstantCompressibilityOilPvt
                                   const Evaluation& pressure) const
     {
         // Eclipse calculates the viscosity in a weird way: it
-        // calcultes the product of B_w and mu_w and then divides the
-        // result by B_w...
+        // calcultes the product of B_o and mu_o and then divides the
+        // result by B_o...
```

Comment:
(I know this is not really the important part of the PR...)

I believe this comment is a bit misleading: probably it never uses viscosity separate from the formation volume factor, so while the effect is as you describe, it is probably not done that way. On the other hand *we* have to do it like that (if we want the same results) since we have a separate interface for viscosity (and replacing that with an interface like "viscosityMultipliedByFormationVolumeFactor" seems too weird).